api/src/main/resources/_core/database/mysql/stored-procedures/sp_mamba_extract_report_metadata.sql

```diff
@@ -45,13 +45,17 @@ BEGIN
                 ELSE
 
                     SET @col_count = 1;
-                    -- SET @column_array = CONCAT('{',@column_array,'}');
+                    SET @table_name = fn_mamba_remove_quotes(@flat_table_name);
+                    SET @current_table_count = 1;
+
                     WHILE @col_count <= @column_keys_array_len
                         DO
                             SELECT fn_mamba_get_array_item_by_index(@column_keys_array, @col_count) INTO @field_name;
                             SELECT fn_mamba_json_value_by_key(@column_array,  @field_name) INTO @concept_uuid;
 
-                            SET @tbl_name = '';
+                            IF @col_count > 100 THEN
```

Comment:
why 100? I thought the limit was 200

Reply:
Was testing out with our tables, guess i forgot to change it back to 200